src/Api/Suppression.php

```diff
@@ -41,8 +42,13 @@ class Suppression
      */
     private $hydrator;
 
-    public function __construct(ClientInterface $httpClient, RequestBuilder $requestBuilder, Hydrator $hydrator)
+    public function __construct($httpClient, RequestBuilder $requestBuilder, Hydrator $hydrator)
     {
+        if (!is_a($httpClient, ClientInterface::class) &&
```

Comment:
Let's change this to `instanceof` instead.

Reply:
Hi @Nyholm , should I do the same for `HttpApi` ?

https://github.com/mailgun/mailgun-php/blob/master/src/Api/HttpApi.php#L47-L53

Reply:
Yes please